src/sass/mixins/_BrandIcon.Mixins.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+// Generate classes for product and document icons
+@mixin brandIconClasses($retina) {
+	@each $icon in $ms-productIcons {
```

Comment:
Good catch. Looks like this file is using tabs instead of spaces. We're going to be adding linting (#869) soon to keep inconsistencies like this from getting checked in.